alws/crud.py

```diff
@@ -450,6 +450,62 @@ async def build_done(
         db.add_all(artifacts)
         db.add(build_task)
         await db.commit()
+    if build_task.arch == 'x86_64' and status == BuildTaskStatus.COMPLETED:
+        src_pkg_name = next(
+            artifact.name for artifact in request.artifacts
+            if artifact.arch == 'src'
+        )
+        # in next commit there will be a response from API
```

Comment:
It seems it's the alpha version. Ping me please when you will be ready for code review :)